docs/BesuProxyModel.md

```diff
@@ -65,4 +66,5 @@ use these steps
 cd $TCF_HOME/docs/dev-environments/besu
 docker-compose down
 ./cleanup.sh
+rm -f $TCF_HOME/blockchain_connector/bookmark $TCF_HOME/examples/apps/generic_client/bookmark
```

Comment:
Please move this command to docker-compose file of Avalon besu and execute it based on a flag

Reply:
@manojgop, leaving it for now. Expect it to be resolved soon as part of re-work on event handling for Ethereum. [Issue](https://github.com/hyperledger/avalon/issues/311) raised here.